htsinfer/get_library_type.py

```diff
@@ -127,13 +127,22 @@ def _evaluate_mate_relationship(
                 self.mapping.library_type.relationship = (
                     StatesTypeRelationship.split_mates
                 )
-        else:
+        elif (
+            self.library_source.file_1.short_name is not None and
+            self.library_source.file_2.short_name is not None
+        ):
             self.mapping.library_type.relationship \
                 = StatesTypeRelationship.not_available
             self.mapping.library_source = self.library_source
             self.mapping.paths = self.path_1, self.path_2
             self.mapping.evaluate()
             self._align_mates()
+        else:
+            self.results.relationship = StatesTypeRelationship.not_available
+            LOGGER.debug(
+                "Library source is not determined, "
+                "mate relationship cannot be inferred by alignment."
+            )
```

Comment:
I think this should rather be a warning? Or does this lead to a warning being raised elsewhere? Anyway, we wanna be consistent, so if other failed "inferences" don't lead to a warning, then this one shouldn't either.

Also, I think the message should include both of the other cases, not just the inability to do alignments. Or just keep it simple and simply say that the mate relationship could not be determined.

Reply:
I thought so too at first, but all the information regarding the process (the seq id format, if alignment is needed etc) are all on the debug level, so getting a warning about the mate relationship determination when running it on info level seems a bit weird, doesn't it?
But yeah, I update the message to say the relationship cannot be determined.